Wox/ViewModel/ResultsViewModel.cs

```diff
@@ -197,8 +199,37 @@ private List<ResultViewModel> NewResults(List<Result> newRawResults, string resu
 
             return results;
         }
+        #endregion
```

Comment:
@jjw24 putting Dependency property in view model is not a very good practice (best way is to put it in a control or behavior).

Reply:
I am not too across dependency property. What do you think @SysC0mp?

Reply:
I think you are right, @theClueless . Should be moved into its own control.